src/main/resources/application.conf.example

```diff
@@ -1,4 +1,15 @@
 mms {
+    application {
+        # if true, respond with 404 to unauthorized requests, even when a resource exists (neither confirming nor denying its existence)
+        glomar-response = true
```

Comment:
should add the extra lines so it can be overwritten with env vars, i think the docker conf is built with a copy of this file